src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  box-sizing: border-box;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__name {
+    color: $main-color;
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 0 16px;
+    font-weight: 400;
+  }
+
+  &__review {
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+    display: flex;
+
+    &__review-rating {
+      color: $main-color;
```

Comment:
The variable `$main-color` is used here. Ensure that this variable is defined in your SCSS files. If it's not defined, the SCSS compilation will fail.